themes/bootstrap3/templates/Recommend/SideFacets.phtml

```diff
@@ -32,14 +46,9 @@
     </div>
   <?php endif; ?>
 <?php endif; ?>
-<?php $extraFilters = isset($this->extraSideFacetFilters) ? $this->extraSideFacetFilters : []; ?>
-<?php $collapsedFacets = $this->recommend->getCollapsedFacets() ?>
-<?php $filterList = array_merge($results->getParams()->getFilterList(true), $extraFilters); ?>
-<?php if (!empty($filterList)): ?>
+<?php if (!empty($this->extraSideFacetFilters)): ?>
```

Comment:
I can't seem to find any situation where extraSideFacetFilters is set. Am I missing something, or is this an obsolete feature? If it's obsolete, can we remove this and also completely eliminate the Recommend/SideFacets/filter-list.phtml file?

Reply:
To answer my own question, it appears that this was only ever used for the special collection side facets, and #1270 rendered this obsolete. I will go ahead and remove it.